Net31Solution/SensorApp/App.xaml

```diff
@@ -0,0 +1,9 @@
+﻿<Application x:Class="SensorApp.App"
```

Comment:
Decompose the code into separate folders at least to support architectural layers. 

Reorganize your classes into Data, Business, UI layers.